README.md

```diff
@@ -51,7 +51,7 @@ The installation of these tools depends on the operating system:
 | `md5sum --version`| `md5sum/sha1sum [VERSION_NUMBER]`  |
 | `xz --version`    | `xz (XZ Utils) [VERSION_NUMBER]`   |
 
-The next step is to clone the [ORT][ort], [ort-config], [ScanCode][scancode] and the [orthw script][orthw] repositories.
```

Comment:
The old version is actually correct, or in you new version you would have to say "repositories of" but I prefer the original sentence

Reply:
Fixed this for you ..